frontend/src/metabase/query_builder/actions/models.js

```diff
@@ -28,8 +28,14 @@ export const onCancelDatasetChanges = () => (dispatch, getState) => {
 
 export const turnQuestionIntoDataset = () => async (dispatch, getState) => {
   const question = getQuestion(getState());
-  const dataset = question.setDataset(true);
-  await dispatch(apiUpdateQuestion(dataset, { rerunQuery: true }));
+  const dataset = await question
+    .setDataset(true)
+    .setDisplay("table")
+    .reduxUpdate(dispatch);
```

Comment:
Not a big deal, but do we really need to use `reduxUpdate` here? We'd want to remove it from the `Question` class eventually, so it'd be nice not to introduce new usages of it

Reply:
removed